data/json/items/comestibles/meat_dishes.json

```diff
@@ -951,7 +951,10 @@
     "type": "COMESTIBLE",
     "id": "deluxe_eggs",
     "name": { "str_sp": "deluxe scrambled eggs" },
-    "conditional_names": [ { "type": "COMPONENT_ID", "condition": "mutant", "name": { "str_sp": "\"deluxe\" scrambled eggs" } } ],
+    "conditional_names": [ 
+     { "type": "COMPONENT_ID", "condition": "mutant", "name": { "str_sp": "\"deluxe\" scrambled eggs" } },
+     { "type": "FLAG", "condition": "CANNIBALISM", "name": { "str_sp": "deluxe eggheads" } }                    
+    ],
```

Comment:
Can you add a strict humanitarian conditional name for something like "birdman breakfast" as well?

Reply:
I like this idea but it would make other names look pale on comprasion

Reply:
No stress.  I'll see if I can think of something that fits the format.

Reply:
Birdman's scrambled eggs (substitute for deluxe)?

Reply:
Yeah, i think that works. Seems like this format would sound nicer for cannibalism too, but idk what word would it be...

Reply:
Well, I think this works largely because "bird" and "eggs" implies that it's from them, not eaten by them.